obspy/io/sac/sactrace.py

```diff
@@ -1100,7 +1100,7 @@ def reftime(self, new_reftime):
     # --------------------------- I/O METHODS ---------------------------------
     @classmethod
     def read(cls, source, headonly=False, ascii=False, byteorder=None,
-             checksize=False, debug_strings=False):
+             checksize=False, debug_strings=False, **kwargs):
```

Comment:
Great :) Public methods (not starting with underscore) should be backwards compatible but just adding a new argument that defaults to the old behavior of course satisfies that constraint.